ibis/file/csv/tests/test_basic.py

```diff
@@ -0,0 +1,116 @@
+import pytest
+
+from pandas.util import testing as tm
+import ibis
+from ibis.file.csv.client import CSVClient, CSVTable
+from ibis.file.client import Database, execute
+
+
+@pytest.fixture
+def transformed(csv):
+
+    # we need to cast to a timestamp type
+    # as we read in as strings
+    closes = csv.csv.close
```

Comment:
Can you call the second `csv` in `csv.csv` something else to indicate it's as directory like `csv_dir` or something like that?

Reply:
done